handlers/bootstrap.go

```diff
@@ -157,13 +155,13 @@ func pullImage(dockerClient *docker.Client) error {
 	return nil
 }
 
-var getRegistrationUrl = func(accountId string, apiClient *client.RancherClient) (string, error) {
+var getRegistrationUrl = func(accountId string, apiClient *client.RancherClient) (string, string, string, error) {
```

Comment:
This is probably picky but this function no longer just returns a URL.  You should rename it to something else like getRegistrationDetails or getRegistratioInfo.